src/content/configuration/module.mdx

```diff
@@ -53,6 +55,28 @@ module.exports = {
 An array of [Rules](#rule) which are matched to requests when modules are created. These rules can modify how the module is created. They can apply loaders to the module, or modify the parser.
 
 
+## `module.unsafeCache`
+
+`boolean` `function (module)`
+
+Cache the resolution of module requests, as well as the module's list of dependencies. Note that this feature can cause problems if the module is moved and should resolve to a different location. By default, this option is enabled only if [`cache`](/configuration/other-options/#cache) is enabled.
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  module: {
+    unsafeCache: false,
+  }
+};
+```
+
+W> In Webpack 5, the default value is `true` if the `cache` option is enabled _and_ the module appears to come from the node_modules directory.
```

Comment:
```suggestion
W> In webpack 5, the default value is `true` if the `cache` option is enabled _and_ the module appears to come from the node_modules directory.
```

Although this documentation is for webpack 5 by default, so the message should be for v4 instead.

Reply:
You had me worried!